src/layers/core/hexagon-layer/hexagon-layer-vertex.glsl

```diff
@@ -22,7 +22,8 @@ uniform float opacity;
 uniform vec3 invisibleColor;
 uniform float radius;
 uniform float angle;
-uniform float enable3d;
+uniform float extruded;
+uniform float radiusScale;
 
 // Lighting constants
 const vec3 ambientColor = vec3(1., 1., 1.);
```

Comment:
We should migrate all 3d layers to use the same lighting code that `PolygonLayer` is using.